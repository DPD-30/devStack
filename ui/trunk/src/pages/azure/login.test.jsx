import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesAppLogin from './login.jsx'

jest.unmock('./login.jsx')


describe('Azure Login ', () => {
    it('Unauthorized should match snapshot', () => {
        const comp = renderToString(<PagesAppLogin />)

        expect(comp).toMatchSnapshot()
    })
})
