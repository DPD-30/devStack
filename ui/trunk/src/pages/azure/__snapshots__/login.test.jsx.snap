// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Azure Login  Unauthorized should match snapshot 1`] = `
<button class="btn btn-secondary"
        data-reactroot
>
  Admin Login
</button>
`;
